app/controllers/purchase-orders/show.js

```diff
@@ -18,7 +18,7 @@ export default Ember.Controller.extend({
       const isTheSameCompany = item.get("company.id") === companyId;
       const matchingOrderItem = orderItems.any(oi => oi.get("item.id") === item.get("id"));
 
-      return isTheSameCompany && !matchingOrderItem
+      return item.get("active") && isTheSameCompany && !matchingOrderItem
```

Comment:
Sorry about this inconvenience. I just updated the code and merge the latest code. Please check it again